src/domain-analysis.js

```diff
@@ -0,0 +1,44 @@
+/* This file contains the code for analyzing number of blog posts from different domains.
+ * The module receives an array of blog urls and returns an array of json object in format
+ * {name: domainName, count: number of frequency}
+*/
+
+
+module.exports.countDomain = function (url) {
+    var domainCounter = []
+
+    for (let i = 0; i < url.length; i++ ){
```

Comment:
This code is really similar to what @smilegodly is doing in #102.  I think you should probably build on what he has, and use the list of URLs his code returns to do the count, then we can store that in Redis later.

Reply:
@humphd  Got it, I will fix it today.

Reply:
@humphd 
We are trying to count the number of blog posts from each domain, not counting number of url domains inside each post, so it should be different from #102. We can simply read from an array of urls and use url.parse to get the hostname and push or add count an object like {name: domainName, count: number}  to a domainList array and return this list.